tests/formats/dataclass/parsers/test_utils.py

```diff
@@ -49,14 +52,65 @@ def test_parse_value_with_ns_map(self, mock_to_python):
             ]
         )
 
+    @mock.patch.object(ParserUtils, "bind_element_wildcard_param")
+    @mock.patch.object(ParserUtils, "find_eligible_wildcard")
+    @mock.patch.object(ParserUtils, "bind_element_param")
+    def test_bind_elements(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5345457)